src/web/docusaurus/docs/getting-started/environment-setup.md

```diff
@@ -244,6 +244,12 @@ Docker builds Telescope's dependencies at launch and keeps them on disk. In some
 
 There are different ways to run the application. By default, [env.development](https://github.com/Seneca-CDOT/telescope/blob/master/config/env.development) will be used. Please read the use cases below to find out what configuration you need to make for different scenarios.
 
+:::caution Important
```

Comment:
Does this work in GitHub?

Reply:
Yea I just checked it's not visible in Markdown do you want me to change it to something else?